src/markdown-pages/collect-data/opentelemetry/set-up-env.mdx

```diff
@@ -0,0 +1,129 @@
+---
+duration: 5
+title: 'Set up your lab environment'
+template: 'GuideTemplate'
+description: 'Spin up your weather forecast application and simulator'
+procIdx: 1
+---
+
+<Callout variant="course" title="lab">
+
+This procedure is part of a lab that teaches you how to instrument your application with OpenTelemetry. If you haven't already, check out the [lab introduction](/collect-data/opentelemetry).
+
+</Callout>
+
+Before you can walk through the lab proper, you need to set up your development environment. Here, you:
```

Comment:
```suggestion
Before you can walk through the lab properly, you need to set up your development environment. Here, you:
```

Reply:
the "lab proper" means the lab, specifically, or without extra context. Setting up part of the environment is a preliminary step and isn't considered part of the "lab proper" here. 

"properly" doesn't really make sense as an adverb in this context as we're not suggesting the walk through the lab properly vs improperly.

I'll think if there's another way to say this though.